addons/packages/packetfence.spec

```diff
@@ -516,9 +516,7 @@ cd $RPM_BUILD_ROOT/usr/local/pf/conf
 #radius sites-enabled symlinks
 #We standardize the way to use site-available/sites-enabled for the RADIUS server
 cd $RPM_BUILD_ROOT/usr/local/pf/raddb/sites-enabled
-ln -s ../sites-available/control-socket control-socket
```

Comment:
The different processes cannot share the same socket so it is specified in the individual files